### PR TITLE
Adapt (inline-)doc regards to new Hydroform branch-install-feature

### PR DESCRIPTION
**Pull request by JeremyHarisch**
**Description**

Changes proposed in this pull request:

- Adapted Docs and Inline-Docs regards [new feature in Hydroform.](https://github.com/kyma-project/hydroform/pull/239)

### CODE REVIEW

**Review by clebs — DISMISSED** [review]

File: cmd/kyma/upgrade/cmd.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ func NewCmd(o *Options) *cobra.Command {
 	cobraCmd.Flags().StringVarP(&o.TLSKey, "tls-key", "", "", "TLS key for the domain used for the upgrade. The key must be a base64-encoded value.")
 	cobraCmd.Flags().StringVarP(&o.Source, "source", "s", DefaultKymaVersion, `Upgrade source.
 	- To use a specific release, write "kyma upgrade --source=1.3.0".
-	- To use the main branch, write "kyma install --source=main".
+	- To use a specific branch (e.g. main-branch), write "kyma install --source=main".
```
Comment (skhalash) [inline cmd/kyma/upgrade/cmd.go:47]:
```suggestion
	- To use a specific branch (e.g. main branch), write "kyma install --source=my-branch-name".
```

File: cmd/kyma/install/cmd.go [hunk 2]
```diff
@@ -85,7 +85,7 @@ The standard installation uses the minimal configuration. The system performs th
 	cobraCmd.Flags().StringVarP(&o.TLSKey, "tls-key", "", "", "TLS key for the domain used for installation. The key must be a base64-encoded value.")
 	cobraCmd.Flags().StringVarP(&o.Source, "source", "s", DefaultKymaVersion, `Installation source.
 	- To use a specific release, write "kyma install --source=1.15.1".
-	- To use the main branch, write "kyma install --source=main".
+	- To use a specific branch (e.g. main-branch), write "kyma install --source=main".
```
Comment (skhalash) [inline cmd/kyma/install/cmd.go:88]:
```suggestion
	- To use a specific branch (e.g. main branch), write "kyma install --source=my-branch-name".
```

File: docs/gen-docs/kyma_install.md [hunk 3]
```diff
@@ -60,7 +60,7 @@ kyma install [flags]
       --profile string         Kyma installation profile (evaluation|production). If not specified, Kyma is installed with the default chart values.
   -s, --source string          Installation source.
                                	- To use a specific release, write "kyma install --source=1.15.1".
-                               	- To use the main branch, write "kyma install --source=main".
+                               	- To use a specific branch (e.g. main-branch), write "kyma install --source=main".
```
Comment (skhalash) [inline docs/gen-docs/kyma_install.md:63]:
```suggestion
                               	- To use a specific branch (e.g. main branch), write "kyma install --source=my-branch-name".
```

File: docs/gen-docs/kyma_upgrade.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ kyma upgrade [flags]
       --profile string         Kyma installation profile (evaluation|production). If not specified, Kyma is installed with the default chart values.
   -s, --source string          Upgrade source.
                                	- To use a specific release, write "kyma upgrade --source=1.3.0".
-                               	- To use the main branch, write "kyma install --source=main".
+                               	- To use a specific branch (e.g. main-branch), write "kyma install --source=main".
```
Comment (skhalash) [inline docs/gen-docs/kyma_upgrade.md:28]:
```suggestion
                               	- To use a specific branch (e.g. main branch), write "kyma install --source=my-branch-name".
```

**Review by skhalash — CHANGES_REQUESTED** [review]

**Review by skhalash — APPROVED** [review]

File: cmd/kyma/install/cmd.go [hunk 5]
```diff
@@ -85,7 +85,7 @@ The standard installation uses the minimal configuration. The system performs th
 	cobraCmd.Flags().StringVarP(&o.TLSKey, "tls-key", "", "", "TLS key for the domain used for installation. The key must be a base64-encoded value.")
 	cobraCmd.Flags().StringVarP(&o.Source, "source", "s", DefaultKymaVersion, `Installation source.
 	- To use a specific release, write "kyma install --source=1.15.1".
-	- To use the main branch, write "kyma install --source=main".
+	- To use a specific branch (e.g. main branch), write "kyma install --source=my-branch-name".
```
Comment (NHingerl) [inline cmd/kyma/install/cmd.go:88]:
```suggestion
	- To use a specific branch (e.g. main branch), write "kyma install --source=<my-branch-name>".
```

As it's a placeholder, let's use brackets

File: cmd/kyma/upgrade/cmd.go [hunk 6]
```diff
@@ -44,7 +44,7 @@ func NewCmd(o *Options) *cobra.Command {
 	cobraCmd.Flags().StringVarP(&o.TLSKey, "tls-key", "", "", "TLS key for the domain used for the upgrade. The key must be a base64-encoded value.")
 	cobraCmd.Flags().StringVarP(&o.Source, "source", "s", DefaultKymaVersion, `Upgrade source.
 	- To use a specific release, write "kyma upgrade --source=1.3.0".
-	- To use the main branch, write "kyma install --source=main".
+	- To use a specific branch (e.g. main branch), write "kyma install --source=my-branch-name".
```
Comment (NHingerl) [inline cmd/kyma/upgrade/cmd.go:47]:
```suggestion
	- To use a specific branch (e.g. main branch), write "kyma install --source=<my-branch-name>".
```

**Review by NHingerl — APPROVED** [review]

**Review by NHingerl — APPROVED** [review]

**Review by skhalash — APPROVED** [review]

**Review by alexandra-simeonova — APPROVED** [review]

**Pull request merged**